hera_qm/tests/test_ant_metrics.py

```diff
@@ -37,26 +37,50 @@ def test_mean_Vij_metrics(self):
                                                 self.ants, self.bls, rawMetric=True)
         self.assertEqual(mean_Vij, {(1, 'y'): 1.0, (1, 'x'): 1.0, (0, 'x'): 1.0,
                                     (0, 'y'): 1.0, (2, 'x'): 1.0, (2, 'y'): 1.0})
+        args = [self.data, self.pols, self.antpols, self.ants, self.bls]
+        zs = uvtest.checkWarnings(ant_metrics.mean_Vij_metrics, args, nwarnings=6,
```

Comment:
Yeah, they're divide by zero. The problem is that the fake data is all ones, and calculating the modified-z for constant data gives nans. I thought about making the data more interesting, but then we would need to figure out what the expected result is to check it. I can change it if you don't think this is a valid test.